README.md

```diff
@@ -65,58 +50,146 @@ To clean up files generated from the build, you can simply run:
 sls offline cleanup
 ```
 
-### Deploy, test, and diagnose your Azure service
+### Deploy Your Function App
 
-1. Deploy your new service to Azure! The first time you do this, you will be asked to authenticate with your Azure account, so the `serverless` CLI can manage Functions on your behalf. Simply follow the provided instructions, and the deployment will continue as soon as the authentication process is completed.
+Deploy your new service to Azure! The first time you do this, you will be asked to authenticate with your Azure account, so the `serverless` CLI can manage Functions on your behalf. Simply follow the provided instructions, and the deployment will continue as soon as the authentication process is completed.
 
-    ```bash
-    serverless deploy
-    ```
+```bash
+$ sls deploy
+```
 
-    > Note: Once you've authenticated, a new Azure "service principal" will be created, and used for subsequent deployments. This prevents you from needing to manually login again. See [below](#advanced-authentication) if you'd prefer to use a custom service principal instead.
+For more advanced deployment scenarios, see our [deployment docs](docs/DEPLOY.md)
 
-2. Invoke a function, in order to test that it works:
+### Test Your Function App
 
-    ```bash
-    serverless invoke -f hello
-    ```
+Invoke your HTTP functions without ever leaving the CLI using:
 
-3. Stream the output logs for your function:
+```bash
+$ sls invoke -f <functionName>
+```
 
-    ```bash
-    serverless logs -f hello
-    ```
- 
-4. Make some code changes, `deploy` again, view logs, etc. and provide us feedback on how to make the experience even better!
+##### Invoke Options
 
-    > Note: If you're working on a single function, you can use the `serverless deploy function -f <function>` command instead of `serverless deploy`, which will simply deploy the specified function instead of the entire service.
+- `-f` or `--function` - Function to Invoke
+- `-d` or `--data` - Stringified JSON data to use as either query params or request body
+- `-p` or `--path` - Path to JSON file to use as either query params or request body
+- `-m` or `--method` - HTTP method for request
 
-If at any point, you no longer need your service, you can run the following command to remove the Azure Functions that were created, and ensure you don't incur any unexpected charges:
+##### Example
+
+After deploying template function app, run
 
 ```bash
-serverless remove
-``` 
+$ sls invoke -f hello '{"name": "Azure"}'
+```
 
-### Advanced Authentication
+### Roll Back Your Function App
+
+To roll back your function app to a previous deployment, simply select a timestamp of a previous deployment and use `rollback` command.
+
+```bash
+# List all deployments to know the timestamp for rollback
+$ sls deploy list
+Serverless:
```

Comment:
this command doesn't seem to work for me - I have done multiple deployments but it returns nothing